procedures/meeting-observers.md

```diff
@@ -0,0 +1,39 @@
+# Meeting observers
+
+## Motivation
+
+There has been some interest in people joining meetings in a pure observational manner. (See [previous discussion on Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Observing.20meetings.3F).) Further discussion on the specifics of a proposal happened on [Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Meeting.20observers) and [in a council meeting](https://github.com/rust-lang/leadership-council/blob/245b22a105546a680aad582cf856311a80e93baf/minutes/sync-meeting/2023-07-20.md).
+
+## Process for meeting observers
+
+### Request for observation
+
+As an initial iteration, any Project member may join the public portion of council meetings to observe. No particular method of communication is required; those interested should either post in the [council zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/392734-council), send an email to `council@rust-lang.org`, or (if necessary) message a council member. At least for now, non-Project members will not be admitted as meeting observers.
+
+### Meeting process
+
+When a meeting is scheduled, those who have requested to observe that meeting will get a calendar invite, in addition to the council. A minutes link will not be shared, but a public agenda link may be.
+
+During the meeting, observers should be mostly non-interactive, unless otherwise noted: no chat messages should be sent, and video and audio should be off. However, there may be times, like during check-ins and check-outs, where observers are encouraged to share their experience in chat. Observers should not record or screenshot the meting. If a meeting has a private session, then observers will either leave or be kicked.
```

Comment:
I wonder if it's better to schedule private meetings as separate meetings, rather than having public and private sections?

I'd also like to register a preference for inviting observers to have their video on. I feel like doing so makes it easier to remember they are fellow humans, and makes it easier to have high trust discussions while being observed. Put another way, I feel like I'd rather have someone sitting in the room with me rather than knowing they are behind a one way mirror. That said, I know other people feel pretty strongly the opposite, and by inviting the observers to join the check in check out we create a chance to build understanding and trust between the council members and observers, so having video the whole time seems less important.

Reply:
Having a separate public and private meeting would make things less complicated re. minutes too.

Reply:
I definitely feel the opposite. I think we should start without video and reevaluate later.

Reply:
I'm going to resolve this - I think we should keep as-is, and revisit with some experience.